HW_1/templates/base.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>
+        {% block title %}
+        Мой магазин
+        {% endblock %}
+    </title>
+</head>
+
+<body>
+    <header>
+        <ul>
+            <li><a href="/">Главная</a></li>
+            <li><a href="/shoes/">Обувь</a></li>
+            <li><a href="/clothes/">Одежда</a></li>
+            <ul>
+                <li><a href="jacket">Куртки</a></li>
```

Comment:
Если повторно нажать на эту ссылку, но произойдет  ошибка, по несуществующей странице а в адресе возникнет `http://127.0.0.1:5000/clothes/jacket/jacket/`. Чтобы такого не было нужно сделать так
```suggestion
                <li><a href="/clothes/jacket/">Куртки</a></li>
```